cmake/toolchain/iar/Kconfig

```
@@ -0,0 +1,40 @@
+# Copyright (c) 2025 IAR Systems AB
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config LD_LINKER_SCRIPT_SUPPORTED
+	default n
+
+choice LINKER_SCRIPT
+	default CMAKE_LINKER_GENERATOR
+endchoice
+
+menu "IAR library options"
+
+config IAR_SEMIHOSTING
+	bool "Use the IAR semihosting implementation."
```

```suggestion
	bool "IAR semihosting implementation."
```
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings
> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

---

Ok!